typescript/sdk/src/metadata/chainMetadataTypes.ts

```diff
@@ -10,6 +10,14 @@ import { ChainMap } from '../types.js';
 
 import { ZChainName, ZNzUint, ZUint } from './customZodTypes.js';
 
+export enum EthJsonRpcBlockParameterTag {
```

Comment:
Can we get this from ethers?

Reply:
![image](https://github.com/user-attachments/assets/9d7f0ed4-1614-46c7-8684-a7f760b37c7f)
unfortunately not :'(
https://github.com/ethers-io/ethers.js/blob/0bfa7f497dc5793b66df7adfb42c6b846c51d794/packages/abstract-provider/src.ts/index.ts#L60